diff --git a/src/vs/workbench/workbench.web.main.ts b/src/vs/workbench/workbench.web.main.ts
index 8306e4e..f4d374d 100644
--- a/src/vs/workbench/workbench.web.main.ts
+++ b/src/vs/workbench/workbench.web.main.ts
@@ -133,7 +133,7 @@ import 'vs/workbench/contrib/debug/browser/extensionHostDebugService';
 import 'vs/workbench/contrib/welcomeBanner/browser/welcomeBanner.contribution';
 
 // Welcome Dialog
-import 'vs/workbench/contrib/welcomeDialog/browser/welcomeDialog.contribution';
+// import 'vs/workbench/contrib/welcomeDialog/browser/welcomeDialog.contribution';
 
 // Webview
 import 'vs/workbench/contrib/webview/browser/webview.web.contribution';
