diff --git a/src/vs/platform/product/common/product.ts b/src/vs/platform/product/common/product.ts
index 3be5dc0..7df0262 100644
--- a/src/vs/platform/product/common/product.ts
+++ b/src/vs/platform/product/common/product.ts
@@ -59,12 +59,12 @@ else {
 	if (Object.keys(product).length === 0) {
 		Object.assign(product, {
 			version: '1.80.0-dev',
-			nameShort: 'Code - OSS Dev',
-			nameLong: 'Code - OSS Dev',
-			applicationName: 'code-oss',
-			dataFolderName: '.vscode-oss',
-			urlProtocol: 'code-oss',
-			reportIssueUrl: 'https://github.com/microsoft/vscode/issues/new',
+			nameShort: 'nopilot Dev',
+			nameLong: 'nopilot Dev',
+			applicationName: 'nopilot',
+			dataFolderName: '.nopilot',
+			urlProtocol: 'nopilot',
+			reportIssueUrl: '',
 			licenseName: 'MIT',
 			licenseUrl: 'https://github.com/microsoft/vscode/blob/main/LICENSE.txt',
 			serverLicenseUrl: 'https://github.com/microsoft/vscode/blob/main/LICENSE.txt'
